certificate-service/app/src/main/java/se/inera/intyg/certificateservice/infrastructure/certificate/persistence/entity/RevokedReason.java

```diff
@@ -0,0 +1,15 @@
+package se.inera.intyg.certificateservice.infrastructure.certificate.persistence.entity;
+
+import lombok.Getter;
+
+@Getter
+public enum RevokedReason {
+  FEL_PATIENT(1), ANNAT_ALLVARLIGT_FEL(2);
```

Comment:
I think we should translate these reasons and use English terms for consistency. It can be mapped in Webcert before calling CS.